app-vim/lightline/lightline-9999.ebuild

```diff
@@ -1,21 +1,22 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
 
 inherit vim-plugin
 
+# LIGHTLINE_COMMIT=
```

Comment:
I prefer that this constant is moved into the `else` block, closer to usage, less places to miss/break.